app/assets/stylesheets/admin.scss

```diff
@@ -96,3 +96,29 @@
 .admin-navigation-links {
   width: 240px;
 }
+
+.config-authentication__row {
+  &--select {
+    display: grid;
+    grid-template-columns: auto 1fr auto;
```

Comment:
Aren't there actually 2 columns?

Reply:
Ah yes. I wrote this initially because I had also include the button on the right that says "Edit", but I ended up removing that markup anyway to simplify this PR. I'll update this.